src/include/storage/data_table.h

```diff
@@ -265,7 +265,10 @@ class DataTable : public AbstractTable {
     default_layout_ = std::shared_ptr<const Layout>(
         new const Layout(schema->GetColumnCount(), type));
   }
-  const Layout &GetDefaultLayout() const;
+
+  const std::shared_ptr<const Layout> GetDefaultLayout() const {
+  	return default_layout_;
+  }
```

Comment:
I'm wondering out of what reasons we change a reference of `default_layout_` to its pointer.

Reply:
Because other getter functions of other objects in `data_table` like  `Index` or `TileGroup` return share_ptr pointer. I didn't understand why it offered the reference of layout.